nautobot/extras/api/serializers.py

```diff
@@ -1021,6 +1032,13 @@ class StatusModelSerializerMixin(serializers.Serializer):
 class TagSerializer(CustomFieldModelSerializer):
     url = serializers.HyperlinkedIdentityField(view_name="extras-api:tag-detail")
     tagged_items = serializers.IntegerField(read_only=True)
+    content_types = ContentTypeField(
+        queryset=ContentType.objects.filter(
+            ModelSubclassesQuery("nautobot.core.models.generics.PrimaryModel").get_query()
+        ),
```

Comment:
```suggestion
        queryset=ModelSubclassesQuery("nautobot.core.models.generics.PrimaryModel").as_queryset
```

Reply:
This code was duplicated as-is from. It appears this function just attempts to account for all potential related classes of a model such as proxy models, etc. https://github.com/Suor/django-cacheops/blob/master/cacheops/query.py#L422-L429